docs/params-configurations.md

```diff
@@ -90,6 +90,10 @@ Or add by yourself. Follow the template in `layouts/partials/socials.html`.
 
 Set your site's start year, then the header will show as **siteStartYear - currentYear (eg: 2016 - 2018)**
 
+### `repository = "BarryYan/barryyan.github.io"`
```

Comment:
Use a more meaningful name? The name `repository` is a little confusing.

Reply:
Or `commentRepo` ? to be normal?

Reply:
I prefer to tell the user what they use. The name of `commentRepo` is still unclear because this theme also supports Disqus.

Reply:
Fine. `utterances` is similar with `comments`,  also is the plugin name. So just use your first suggestion `utterancesRepo`.

Reply:
cc @g1eny0ung